src/overlays/actors/ovl_Dm_Stk/z_dm_stk.c

```diff
@@ -554,27 +732,32 @@ void func_80AA076C(DmStk* this, GlobalContext* globalCtx) {
             break;
     }
 
-    if (this->unk_2E0 == 0) {
+    if (this->animationId == SK_ANIMATION_SHAKE_HEAD) {
         if (Animation_OnFrame(&this->skelAnime, 8.0f) || Animation_OnFrame(&this->skelAnime, 17.0f)) {
             Actor_PlaySfxAtPos(&this->actor, NA_SE_EN_STALKIDS_MASK_OFF);
         }
 
         if (Animation_OnFrame(&this->skelAnime, 28.0f)) {
             Actor_PlaySfxAtPos(&this->actor, NA_SE_EN_STALKIDS_RIDE);
         }
-    } else if (this->unk_2E0 == 71) {
+    } else if (this->animationId == SK_ANIMATION_LAUGH_AFTER_SNIFF) {
         if (Animation_OnFrame(&this->skelAnime, 2.0f) || Animation_OnFrame(&this->skelAnime, 6.0f) ||
             Animation_OnFrame(&this->skelAnime, 12.0f) || Animation_OnFrame(&this->skelAnime, 18.0f)) {
             Actor_PlaySfxAtPos(&this->actor, NA_SE_EN_STALKIDS_MASK_OFF);
         }
-    } else if ((this->unk_2E0 == 70) &&
+    } else if ((this->animationId == SK_ANIMATION_SNIFF) &&
                (Animation_OnFrame(&this->skelAnime, 16.0f) || Animation_OnFrame(&this->skelAnime, 23.0f))) {
         Actor_PlaySfxAtPos(&this->actor, NA_SE_EN_STALKIDS_NOSE);
     }
 }
 
-void func_80AA09DC(DmStk* this, GlobalContext* globalCtx) {
-    static s32 D_80AA3CBC = 0;
+/**
+ * Handles sound effects for the intro cutscene at the top of the Clock Tower. Specifically,
+ * it handles the variation of the cutscene that plays the second or later time the player
+ * reaches the top of the Clock Tower, which is slightly shorter.
+ */
+void DmStk_PlaySfxForClockTowerIntroCutsceneSubsequentTimes(DmStk* this, GlobalContext* globalCtx) {
```

Comment:
I think just "FirstVersion" and "SecondVersion" would be good enough since there's a comment here anyway.

Reply:
(Or something even shorter, these function names are getting into `Camera2_Talk_GetGoalDirectionAngleXYZAndSetDistance_BetweenSpeakerToListener` territory.)